doc/src/site/sphinx/deployment/load_mojo_pipeline.rst

```diff
@@ -0,0 +1,167 @@
+
+        .. code:: bash
+
+            ./bin/pysparkling --jars license.sig
+
+
+At this point, you should have Spark interactive terminal where we can carry out predictions.
+For productionalizing the scoring process, we can use the same configuration,
+except instead of using Spark shell, we would submit the application using ``./bin/spark-submit``.
+
+Now Load the MOJO as:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.models.H2OMOJOPipelineModel
+            val settings = H2OMOJOSettings()
+            val mojo = H2OMOJOPipelineModel.createFromMojo("file:///path/to/the/pipeline_mojo.zip", settings)
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            from pysparkling.ml import H2OMOJOPipelineModel
+            settings = H2OMOJOSettings()
+            mojo = H2OMOJOPipelineModel.createFromMojo("file:///path/to/the/pipeline_mojo.zip", settings)
+
+In the examples above ``settings`` is an optional argument. If it's not specified, the default values are used.
+
+Prepare the dataset to score on:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            val dataFrame = spark.read.option("header", "true").csv("file:///path/to/the/data.csv")
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            dataFrame = spark.read.csv("file:///path/to/the/data.csv", header=True)
+
+And finally, score the mojo on the loaded dataset:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            val predictions = mojo.transform(dataFrame)
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            predictions = mojo.transform(dataFrame)
+
+We can select the predictions as:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            predictions.select("prediction")
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            predictions.select("prediction")
+
+The output data frame contains all the original columns plus the prediction column which is by default named
+``prediction``. The prediction columns contains all the prediction detail. Its name can be modified via the ``H2OMOJOSettings``
+object.
+
+Customizing the MOJO Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can configure the output and format of predictions via the H2OMOJOSettings. The available options are
+
+- ``predictionCol`` - Specifies the name of the generated prediction column. Default value is `prediction`
+- ``detailedPredictionCol`` - Specified the name of the generated detailed prediction column. The detailed prediction column,
```

Comment:
Are all of the properties relevant to the pipeline mojo? E.g. `withContributions`, `withDetailedPredictionCol`, `withDetailedPredictionCol `

Reply:
Good point, will verify which ones are related

Reply:
Thanks, this was really good catch. The valid options are only:

`predictionCol`, `convertUnknownCategoricalLevelsToNa`, `convertInvalidNumbersToNa` and `namedMojoOutputColumns`